### ISSUE TITLE
'not' syntax for cell filtering

### ISSUE THREAD

**Issue by JohnGriffiths** [jdfreder/notebook_cherry_picker#3]
I've been trying use a 'not' syntax for the cherry picker expression and am getting funny results for what I would have thought should be the correct code. 

Suppose I have succesfully set up TAGs 1-3, and used the cell toolbar to tick them as desired. 

What would be the syntax for removing cells tagged with TAG1?

This doesn't seem to be working for me

```
' --CherryPickingPreprocessor.expression="not TAG1"'
```

Is that incorrect?

Ta.

**Comment by jdfreder** [comment]
Hey @JohnGriffiths 

Thanks for reporting this; it looks like a change in nbconvert broke things.  I'll ping back when I have a fix.

**Comment by jdfreder** [comment]
Until https://github.com/ipython/ipython/pull/5670 is merged, it looks like this will be broken.  I'll work on a look around.

**Comment by JohnGriffiths** [comment]
Thanks very much. 

Sounds like the problem is having the .py files in the CWD?

 In actual fact I would prefer to have them in a separate directory anyway. But I couldn't figure out how to do this. Is it possible?

**Comment by jdfreder** [comment]
I don't think we add the profile directory to the Python path, otherwise I'd suggest just placing the files in your profile directory...  I'll open an issue suggesting that as a new feature.  For now, you can just places the writer and preprocessor py files in your ipython/ folder (I know, it's a crappy workaround, but it will work). 

It's been a while since I've looked at this.  I forgot that it exported notebooks instead of the output specified by `--to` (it bypasses nbconvert's internal export).  I'm not sure if anyone else uses this cherry picker as is, but I have no problem changing that for you so it outputs to the format specified by `--to` for you.
